Userland/Libraries/LibGfx/Filters/InvertFilter.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, Musab Kılıç <musabkilic@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Filters/Filter.h>
+
+namespace Gfx {
+
+class InvertFilter : public Filter {
+public:
+    InvertFilter() { }
+    virtual ~InvertFilter() { }
+
+    virtual char const* class_name() const override { return "InvertFilter"; }
+
+    virtual void apply(Bitmap& target_bitmap, IntRect const& target_rect, Bitmap const& source_bitmap, IntRect const& source_rect) override
+    {
+        // source_rect should be describing the pixels that can be accessed
+        // to apply this filter, while target_rect should describe the area
+        // where to apply the filter on.
```

Comment:
Just my opinion, but I don't think this comment is really necessary. `source_rect` and `target_rect` are descriptive enough, this just seems redundant.